src/pages/packer/plugins/[...page].tsx

```diff
@@ -104,7 +104,16 @@ export async function getStaticProps({ params, ...ctx }) {
 			levelButtonProps: {
 				levelUpButtonText: `${productData.name} Home`,
 			},
-			menuItems: navData,
+			menuItems: [
+				{
+					title: 'Plugins',
+					fullPath: `/${productData.slug}/plugins`,
+					theme: productData.slug,
+				},
+				// Remove the authored title element and "Overview" link, these
+				// would be redundant with the styled overview item that we add.
+				...navData.slice(2),
```

Comment:
🙋🏻 Will this break when these items are changed at the source?

Reply:
Great callout, they would 💖 I've updated to use a shared `addBrandedOverviewSidebarItem`, which is significantly more robust in how it checks for and removes potentially redundant items before adding the "branded" overview item:
- If first item is an "Overview" item, removes it
- Else if first item is a heading `MenuItem`, and second item is and "Overview" item, removes them both
- Else leaves items unchanged

Reply:
That sounds sweet! I'll take a look at that 🤩